pkg/proxy/proxyinfo.go

```diff
@@ -54,6 +55,9 @@ func GetProxyInfo(kubeClient kube.CLIClient, istioNamespace string) (*[]istioVer
 			})
 		}
 	}
+	sort.SliceStable(pi, func(i, j int) bool {
```

Comment:
Should we sort `/debug/syncz` to avoid sorting its results in different places?

Reply:
Good idea 💡